src/Common/src/TypeSystem/Common/TargetDetails.cs

```diff
@@ -54,6 +70,11 @@ public TargetOS OperatingSystem
             get;
         }
 
+        public TargetApplicationModel ApplicationModel
+        {
+            get; private set;
```

Comment:
Is the `private set` part really needed? (It is not needed for other properties...)